backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Preferences.java

```diff
@@ -34,18 +34,25 @@
 
 public class Lwjgl3Preferences implements Preferences {
 	private final Properties properties = new Properties();
-	private final FileHandle file;
+	private FileHandle file;

```

Comment:
I'm not clear why this has its own `FileHandle` but never appears to use it outside of one local usage in a constructor. I might be missing something, but could this be made local? I believe IDEA has an automatic inspection for that.